apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -260,6 +262,18 @@ class ScriptEditor extends React.Component {
       is_migrated: this.props.isMigrated
     };
 
+    if (this.state.weeklyInstructionalMinutes) {
+      dataToSave.weekly_instructional_minutes =
+        parseInt(this.state.weeklyInstructionalMinutes) || null;
+    }
+    if (dataToSave.show_calendar && !dataToSave.weekly_instructional_minutes) {
```

Comment:
Same wonder as above should we check that it weekly_instructional_minutes is greater than zero

Reply:
Thinking a little bit more here if you move this check above when the dataToSave gets set up and just use the state instead of dataToSave for the condition then we can just put weekly_instructional_minutes back in the dataToSave because it should never violate that condition